integration/log_rotation_test.go

```diff
@@ -0,0 +1,153 @@
+package integration
```

Comment:
Could we use a build tag here to avoid running the test on Windows (where it would always fail to my understanding)? Users may want to run the tests natively, i.e., outside of our build container.